packages/neon/neon_notifications/lib/neon_notifications.dart

```diff
@@ -21,7 +21,9 @@ part 'options.dart';
 part 'pages/main.dart';
 
 class NotificationsApp extends AppImplementation<NotificationsBloc, NotificationsAppSpecificOptions>
-    implements NotificationsAppInterface<NotificationsBloc, NotificationsAppSpecificOptions> {
+    implements
+        // ignore: avoid_implementing_value_types
```

Comment:
This is really ugly. Could this be moved up or just be ignored for the whole file?